portal/portal.py

```diff
@@ -89,6 +63,35 @@ def delete(id):
     cur.close()
     return redirect(url_for('portal.home'))
 
+
 @bp.route("/student")
 def student():
     return render_template("student-home.html")
+
+
+@bp.route("/create", methods=['GET', 'POST'])
```

Comment:
the create courses route should be before edit and delete rout, if its the end I am seeing it at the bottom.